bitrise.yml

```diff
@@ -218,7 +316,8 @@ app:
     - opts:
         is_expand: false
       SKIP_APOLLO_CODEGEN: '1'
+
 meta:
   bitrise.io:
-    machine_type_id: g2.12core
+    machine_type_id: g2-m1.8core
```

Comment:
In my opinion there are workflows that could use a lower end machine. For example, the unit tests run in parallel with UI tests, we don't need them to run super fast as the build will wait for the other. We can set a meta with a 4core for this test suite by adding this meta code within the workflow. Each workflow can use a different one as long as we can use Intel machines to build and test.
Same for the deploy step at the end. I have not seen a big difference in using 4core and 8core but credits are x6 per minute. It would be a good optimization to reduce that a bit.

Reply:
@isabelrios Oh great! I did not know bitrise supported machines per workflows. I will look at this and I agree.

Reply:
Yes, that's possible :) see on firefox bitrise.yml file as an example: https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml#L381, that meta is for a workflow that runs in parallel with others.